src/broker/cas_execute.c

```diff
@@ -1118,6 +1118,14 @@ ux_cgw_prepare (char *sql_stmt, int flag, char auto_commit_mode, T_NET_BUF * net
 	  err_code = ERROR_INFO_SET (db_error_code (), DBMS_ERROR_INDICATOR);
 	  goto prepare_error;
 	}
+
+      cgw_set_stmt_attr (srv_handle->cgw_handle->hstmt, SQL_ATTR_CURSOR_TYPE, (SQLPOINTER) SQL_CURSOR_STATIC,
+			 SQL_IS_INTEGER);
+      if (err_code < 0)
```

Comment:
this code block would be better to be moved to the front of **cgw_set_stmt_attr(...)**

Reply:
fixed it